omftools/resources/tags.csv

```diff
@@ -48,13 +48,13 @@
 "bt",0,"Dark tint, used by shadow HAR"
 "by",0,
 "cf",0,"Only used by shadow scrap, works with 'bm' tag to walk to far corner of arena"
-"cg",0,
-"cl",0,
+"cg",0,"Triggers behaviors when touching the ground. Used by Katana to trigger landing recovery. (Enabling the d<> tag)"
+"cl",0,"Triggers the next animation when touching the ground. Used by Pyros to proceed with scrap."
 "cp",0,"If hit collision succeeds, perform hit pause"
-"cw",0,
+"cw",0,"Enables a wall collision for the enemy and triggers as if this move had hit. Used in Flail's scrap to automatically trigger jn"
```

Comment:
Please keep these short, under 90 characters (or so) if possible. Just a short description of what it does. Otherwise the outputs tend to be mess in the strings tool and in the C file. More detailed descriptions should be in the wiki.